docs: Broken links found

# Summary

| Status        | Count |
|---------------|-------|
| 🔍 Total      | 714   |
| ✅ Successful | 693   |
| ⏳ Timeouts   | 1     |
| 🔀 Redirected | 0     |
| 👻 Excluded   | 14    |
| ❓ Unknown    | 0     |
| 🚫 Errors     | 6     |

## Errors per input

### Errors in conferences/2025/general.json

* [TIMEOUT] [https://www.coderful.io/](https://www.coderful.io/) | Timeout

### Errors in conferences/2025/opensource.json

* [404] [https://2025.osday.dev/code-of-conduct](https://2025.osday.dev/code-of-conduct) | Error (cached)

### Errors in conferences/2025/devops.json

* [404] [https://2025.osday.dev/code-of-conduct](https://2025.osday.dev/code-of-conduct) | Network error: Not Found
* [ERROR] [https://qconlondon.com/medialisting2025](https://qconlondon.com/medialisting2025) | Network error: error sending request for url (https://qconlondon.com/medialisting2025) Maybe a certificate error?

### Errors in conferences/2025/javascript.json

* [404] [https://2025.osday.dev/code-of-conduct](https://2025.osday.dev/code-of-conduct) | Error (cached)

### Errors in conferences/2025/rust.json

* [404] [https://forms.gle/rKY6SdZPFn3Ua8vF7](https://forms.gle/rKY6SdZPFn3Ua8vF7) | Network error: Not Found

### Errors in conferences/2025/identity.json

* [403] [https://internetidentityworkshop.com/](https://internetidentityworkshop.com/) | Network error: Forbidden
[Full Github Actions output](https://github.com/tech-conferences/conference-data/actions/runs/13468884972?check_suite_focus=true)

--- a/conferences/2025/devops.json
+++ b/conferences/2025/devops.json
@@ -66,7 +66,6 @@
     "country": "Italy",
     "online": true,
     "locales": "EN",
-    "cocUrl": "https://2025.osday.dev/code-of-conduct",
     "cfpUrl": "https://sessionize.com/open-source-day-2025",
     "cfpEndDate": "2025-01-15",
     "twitter": "@schrodinger_hat"

--- a/conferences/2025/javascript.json
+++ b/conferences/2025/javascript.json
@@ -106,7 +106,6 @@
     "country": "Italy",
     "online": true,
     "locales": "EN",
-    "cocUrl": "https://2025.osday.dev/code-of-conduct",
     "cfpUrl": "https://sessionize.com/open-source-day-2025",
     "cfpEndDate": "2025-01-15",
     "twitter": "@schrodinger_hat"

--- a/conferences/2025/opensource.json
+++ b/conferences/2025/opensource.json
@@ -70,7 +70,6 @@
     "country": "Italy",
     "online": true,
     "locales": "EN",
-    "cocUrl": "https://2025.osday.dev/code-of-conduct",
     "cfpUrl": "https://sessionize.com/open-source-day-2025",
     "cfpEndDate": "2025-01-15",
     "twitter": "@schrodinger_hat"

--- a/conferences/2025/rust.json
+++ b/conferences/2025/rust.json
@@ -9,8 +9,6 @@
     "online": true,
     "locales": "EN",
     "cocUrl": "https://www.rustinparis.com/code-of-conduct",
-    "cfpUrl": "https://forms.gle/rKY6SdZPFn3Ua8vF7",
-    "cfpEndDate": "2024-11-30",
     "twitter": "@xperhub"
   },
   {

